src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Christoffel.hpp

```diff
@@ -18,8 +18,8 @@ namespace GeneralizedHarmonic {
  *
  * \details
  * If \f$ \Phi_{kab} \f$ is the generalized harmonic spatial derivative
- * variable \f$ \Phi_{kab} = \partial_k \psi_{ab}\f$ and \f$\gamma^{ij}\f$
```

Comment:
For what it's worth, I think it was a big mistake for the GH paper to have introduced g and psi instead of gamma and g. I understand why it was done, but nevertheless it's never a good idea to introduce your own notation for something that's already established in the literature.

Reply:
The fixup pushed addresses all but this, as I wasn't sure if a consensus was reached.